models/topicsModel.js

```diff
@@ -1,16 +1,25 @@
 const db = require("../db/connection");
-const fs = require("fs/promises")
+const fs = require("fs/promises");
 
+exports.getAllTopics = () => {
+  return db.query("SELECT * FROM topics").then((topics) => {
+    return topics.rows;
+  });
+};
 
-exports.getAllTopics = () =>{
-    return db.query("SELECT * FROM topics").then((topics)=>{
-        return topics.rows
-    })
-}
+exports.getAllApis = () => {
+  return fs.readFile("./endpoints.json", "utf-8").then((apis) => {
+    return JSON.parse(apis);
+  });
+};
```

Comment:
This totally works - but you could just require in your endpoints file and parse it from there